### PR TITLE
Enhance the aks user guide.

### PR DESCRIPTION
**Pull request by galiacheng**
Changes:

1. Apply Reza‘s review comments.
2. Add a field for weblogic cup repo in **Image selection** section.
3. Remove T3 related text as we have disabled the T3 setting from offer UI.

Signed-off-by: galiacheng <haixia.cheng@microsoft.com>

 On branch aks-userguide
 Changes to be committed:
	modified:   documentation/3.3/content/samples/azure-kubernetes-service/includes/aks-value-prop.txt
	modified:   documentation/3.3/content/userguide/aks/_index.md
	modified:   documentation/staging/content/samples/azure-kubernetes-service/includes/aks-value-prop.txt
	modified:   documentation/staging/content/userguide/aks/_index.md

### CODE REVIEW

**Comment by galiacheng** [comment]
Hello @edburns  @m-reza-rahman @rosemarymarano may I have your attention to this pr?

File: documentation/3.3/content/userguide/aks/_index.md [hunk 1]
```diff
@@ -80,23 +78,24 @@ In this section, you can configure some options about the AKS which will run Web
...
-| Password for Oracle Single Sign-On authentication | The password for that account. | 
-| Confirm password | Re-enter the value of the preceding field. |
+| Password for Oracle Single Sign-On authentication | The password for that account. |
+| Confirm password | Re-enter the value of the preceding field. | If 'Yes' is selected, the deployment process will pull from the CPU WebLogic Server image repository in the OCR. If 'No' is selected the deployment process will pull from the WebLogic Server image repository in OCR. |
+| Is the specified SSO account associated with an active Oracle support contract? | If set to **Yes**, you must accept the license agreement in `middleware/weblogic_cpu` repository. If set to **No**, you must accept the license agreement in `middleware/weblogic`. Steps to accept the license agreement: log on to [Oracle Container Registry](https://container-registry.oracle.com/); navigate to repository `middleware/weblogic_cpu` and `middleware/weblogic`; accept license agreement. See this [document](https://aka.ms/wls-aks-ocr-doc) for more information. |
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:98]: license agreement in `middleware/weblogic_cpu`  -> license agreement in the `middleware/weblogic_cpu` 
log on to -> log in to the
navigate to repository -> navigate to the `middleware/weblogic_cpu` and `middleware/weblogic` repositories
accept license agreement -> accept the license agreement

File: documentation/3.3/content/userguide/aks/_index.md [hunk 2]
```diff
@@ -105,18 +104,18 @@ In this section you can deploy a Java EE Application along with the WebLogic Ser
...
 
-Select **Yes** or **No** for the option **Configure WebLogic Server Administration Console, Remote Console, cluster and custom T3 channel to use HTTPS (Secure) ports, with your own TLS/SSL certificate.** based on your preference. If you select **No**, you don't have to provide any details, and can proceed by selecting **Next : Networking** >. If you select **Yes**, you can choose to provide the required configuration details by either uploading existing keystores or by using keystores stored in Azure Key Vault.
+Select **Yes** or **No** for the option **Configure WebLogic Server Administration Console, Remote Console, and cluster to use HTTPS (Secure) ports, with your own TLS/SSL certificate.** based on your preference. If you select **No**, you don't have to provide any details, and can proceed by selecting **Next : Networking** >. If you select **Yes**, you can choose to provide the required configuration details by either uploading existing keystores or by using keystores stored in Azure Key Vault.
```
Comment (rosemarymarano) [inline documentation/3.3/content/userguide/aks/_index.md:118]: selecting **Next : Networking** >. -> selecting **Next : Networking**. (remove the angle bracket)

**Review by rosemarymarano — COMMENTED** [review]
Few edits

File: documentation/3.3/content/userguide/aks/_index.md (hunk 2 above)
Comment (galiacheng) [reply documentation/3.3/content/userguide/aks/_index.md]: See https://github.com/oracle/weblogic-kubernetes-operator/pull/2746/commits/fc56f0c0ed55c34ff3e80d6232e8f5f03d84e508

File: documentation/3.3/content/userguide/aks/_index.md (hunk 1 above)
Comment (galiacheng) [reply documentation/3.3/content/userguide/aks/_index.md]: See https://github.com/oracle/weblogic-kubernetes-operator/pull/2746/commits/fc56f0c0ed55c34ff3e80d6232e8f5f03d84e508

File: documentation/3.3/content/userguide/aks/_index.md [hunk 3]
```diff
@@ -80,23 +78,24 @@ In this section, you can configure some options about the AKS which will run Web
...
 | Please provide the image path | This option is shown only if **Use a pre-existing WebLogic Server Docker image from Oracle Container Registry?** is set to **No**. If visible, the value must be a fully qualified Docker tag of an image within the specified ACR. |
 | Username for Oracle Single Sign-On authentication | The Oracle Single Sign-on account user name for which the Terms and Restrictions for the selected WebLogic Server image have been accepted. |
-| Password for Oracle Single Sign-On authentication | The password for that account. | 
-| Confirm password | Re-enter the value of the preceding field. |
+| Password for Oracle Single Sign-On authentication | The password for that account. |
+| Confirm password | Re-enter the value of the preceding field. | If 'Yes' is selected, the deployment process will pull from the CPU WebLogic Server image repository in the OCR. If 'No' is selected the deployment process will pull from the WebLogic Server image repository in OCR. |
```
[the remark this answers is not in the document]
Comment (galiacheng) [reply documentation/3.3/content/userguide/aks/_index.md]: See https://github.com/oracle/weblogic-kubernetes-operator/pull/2746/commits/fc56f0c0ed55c34ff3e80d6232e8f5f03d84e508

File: documentation/3.3/content/userguide/aks/_index.md [hunk 4]
```diff
@@ -80,23 +78,24 @@ In this section, you can configure some options about the AKS which will run Web
...
+|Enable Container insights| If selected, configure the necessary settings to integrate with Container insights. Container insights gives you performance visibility by collecting memory and processor metrics from controllers, nodes, and containers that are available in Kubernetes through the Metrics API. Container logs are also collected. Metrics are written to the metrics store and log data is written to the logs store associated with your Log Analytics workspace. For more information, see [Container insights overview](https://aka.ms/wls-aks-container-insights).|
+|Create Persistent Volume using Azure File share service|If selected, an Azure Storage Account and an Azure Files share will be provisioned; static persistent volume with the Azure Files share will be mounted the nodes of the AKS cluster. For more information, see [Oracle WebLogic Server persistent storage]({{< relref "/userguide/managing-domains/persistent-storage/_index.md" >}}) and [persistent volume with Azure Files share on AKS](https://docs.microsoft.com/azure/aks/azure-files-volume).|
```
[the remark this answers is not in the document]
Comment (galiacheng) [reply documentation/3.3/content/userguide/aks/_index.md]: See https://github.com/oracle/weblogic-kubernetes-operator/pull/2746/commits/fc56f0c0ed55c34ff3e80d6232e8f5f03d84e508

**Comment by galiacheng** [comment]
Hello @rosemarymarano I have applied your comments to the PR, please have a look at your convenience. Thank you!

**Review by rosemarymarano — APPROVED** [review]

**Review by edburns — APPROVED** [review]

**Pull request merged**